src/sdl/sdl_system.c

```diff
@@ -315,7 +315,7 @@ ALLEGRO_SYSTEM_INTERFACE *_al_sdl_system_driver(void)
       return vt;
 
    vt = al_calloc(1, sizeof *vt);
-   vt->id = AL_ID('S', 'D', 'L', '2');
```

Comment:
I'm kind of partial towards this macro, could we use it for everything (i.e. use it as the values in the enum, I'm fine with the enum existing).